monkey/infection_monkey/utils/timer.py

```diff
@@ -0,0 +1,34 @@
+import time
+
+
+class Timer:
```

Comment:
Looks so simple that it's a wonder that it isn't a built-in yet.

Reply:
Yeah, I looked around. There are timer objects in Threading, but they're used to schedule tasks. I could have implemented the same functionality that way, but then that's effectively another thread to manage.